### PR TITLE
DM-6999: Logging framework migration

### PR DESCRIPTION
**Pull request by hchiang2**
Change back to use pexLog.Debug directly to facilitate logging framework migration

### CODE REVIEW

**Comment by isullivan** [comment]
The code changes all look fine. Could you add to the commit message the ticket number that will remove pexLog permanently?

File: python/lsst/meas/extensions/psfex/psfexPsfDeterminer.py [hunk 1]
```diff
@@ -154,6 +155,7 @@ def determinePsf(self, exposure, psfCandidateList, metadata=None, flagKey=None):
         normalizeResiduals = lsstDebug.Info(__name__).normalizeResiduals
                                                             # Normalise residuals by object amplitude
 
+        debugLog = pexLog.Debug("meas.algorithms.psfDeterminer")
```
Comment (r-owen) [inline python/lsst/meas/extensions/psfex/psfexPsfDeterminer.py]: Why did you add this debug log? This task already has a log attribute and the existing code used it.

File: python/lsst/meas/extensions/psfex/psfexPsfDeterminer.py [hunk 2]
```diff
@@ -175,7 +177,7 @@ def determinePsf(self, exposure, psfCandidateList, metadata=None, flagKey=None):
                 if psfCellSet:
                     psfCellSet.insertCandidate(psfCandidate)
             except Exception, e:
-                self.log.log(-2, "Skipping PSF candidate %d of %d: %s" % (i, len(psfCandidateList), e))
+                debugLog.debug(2, "Skipping PSF candidate %d of %d: %s" % (i, len(psfCandidateList), e))
```
Comment (r-owen) [inline python/lsst/meas/extensions/psfex/psfexPsfDeterminer.py]: What does the first argument `2` mean? (Also, why the % formatting?) I would expect to see:

    self.log.debug("Skipping PSF candidate %d of %d: %s", i, len(psfCandidateList), e)

**Comment by hchiang2** [comment]
@r-owen  Originally I thought this was a special case so I made it use `pex.logging.Debug` directly before we clean all pex.logging.Debug/Trace in a later ticket.  At a closer look, one debug message is actually a warning message. Was pex.logging debug level of 1 used as warnings at one point long time ago?   (Or probably just an bug.)  In a new commit, I raised the warning message to the WARN level. And then corrected other logging calls to use `log`. May you please take another look and see if you agree with these changes?

**Comment by r-owen** [comment]
I'll have a look. I also posted a table of old vs new log levels to DM-6999 as a comment

File: python/lsst/meas/extensions/psfex/psfexPsfDeterminer.py [hunk 3]
```diff
@@ -186,8 +186,7 @@ def determinePsf(self, exposure, psfCandidateList, metadata=None, flagKey=None):
             sizes[i] = rmsSize
 
         if self.config.kernelSize >= 15:
-            debugLog.debug(1, \
-                "WARNING: NOT scaling kernelSize by stellar quadrupole moment, but using absolute value")
+            self.log.warn("NOT scaling kernelSize by stellar quadrupole moment, but using absolute value")
```
Comment (r-owen) [inline python/lsst/meas/extensions/psfex/psfexPsfDeterminer.py]: This seems like a sensible change to me

**Comment by r-owen** [comment]
I'm not an expert on the code in question, but your changes look reasonable to me. I suspect many more such decisions will be required due to lack of trace levels in the log package.
